wiki/Guides/Getting_Your_Map_Modded/en.md

```diff
@@ -1,38 +1,43 @@
+3. Read the queue's rules, if you or your map does not meet the requirements listed there, then unfortunately you'll have to find another queue to post in.
+4. Read the modder's preferences listed in the queue (if any), it may save both you and the creator of the queue a lot of time.
+5. Click the `go to last post` button at the bottom of the screen. This is represented by the double right arrow icon.
+6. Type what sort of request you would like, along with a link to your beatmap and any other information the modder has asked you to provide, into the `New reply` box.
+7. Click `Post` when you are finished.
 
-## Modding queues
+It is generally acceptable to post in a few modding queues at a time, so feel free to repeat this process if necessary, or if you want to boost your chances of receiving a mod. Keep in mind, just because you have posted in a modding queue, does not mean the modder is obligated to mod your map, they will mod it if they want to.
+
+It is worth noting that you should avoid posting in Beatmap Nominator's queues until your mapping is at a near rankable level (unless they have specified they are looking for newer mappers), as they will likely always ignore your requests otherwise. You are much better off finding normal modders to mod your maps when you are still learning how to map.
+
+## #modreqs
 
-The [Modding Queues](https://osu.ppy.sh/community/forums/60) section of the forum contains Modding Queues (hard to comprehend, we know). Modding Queues are places where you post maps, and the creator of the thread may eventually mod them. Each thread has specific rules, so follow the thread's rules to get your map modded.
+#modreqs is a channel in the osu! chat where players can request mods on their maps by linking them with the `/np` command, along with a mod request type. While it is not the most effective method of finding mods, modders and Beatmap Nominators are often on the lookout for good maps being linked in this channel to mod.
 
-## Asking other users for a M4M
+#modreqs does not have the same functionality and userbase that it has had in the past, but it is still a useful method for obtaining mods, so it is worth a shot. While it is still full of normal mod requests, these days it is also commonly used for the other request types mentioned above in [types of mods](/wiki/Guides/Getting_Your_Map_Modded#types_of_mods).
 
-A mod4mod, or *M4M* for short, is where you mod somebody else's map if they mod yours. This is a very effective method of getting mods. It's particularly useful if you combine it with #modreqs; if somebody posts a mod request there, simply ask them if they're willing to do a mod4mod. Often they'll say "yes." It takes effort, but is well worth the mod you receive and the experience you gain from modding someone else's map.
+*Warning: Do not spam requests in this channel, post a mod request in #modreqs once every hour or so.*
```

Comment:
Put this warning right below `## #modreqs` ([relevant ASC section](https://github.com/ppy/osu-wiki/blob/master/wiki/Article_styling_criteria/Formatting/en.md#notice))